enterprise/server/raft/store/store.go

```diff
@@ -271,6 +274,9 @@ func NewWithArgs(env environment.Env, rootDir string, nodeHost *dragonboat.NodeH
 		s.driverQueue = driver.NewQueue(s, gossipManager, nhLog, apiClient, clock)
 	}
 	s.deleteSessionWorker = newDeleteSessionsWorker(clock, s)
+	if *zombieNodeScanInterval != 0 {
+		s.replicaJanitor = newReplicaJanitor(clock, s)
+	}
```

Comment:
It can be a little error prone to have stuff only set sometimes -- easy to hit NPEs. Would it be clearer to always set the janitor but just have it not do anything if *zombieNodeScanInterval == 0?